x-pack/plugins/cases/public/containers/use_get_case.test.tsx

```diff
@@ -36,13 +37,13 @@ describe('useGetCase', () => {
     });
   });
 
-  it('calls getCase with correct arguments', async () => {
-    const spyOnGetCase = jest.spyOn(api, 'getCase');
+  it('calls resolveCase with correct arguments', async () => {
```

Comment:
Why we replaced the `getCase` tests?

Reply:
@cnasikas It is because I created a new function `resolveCase` in the api module to call _/resolve_, which is the function used by the hook now. The `getCase` function is not used, but I did not remove it as the api endpoint is not removed either. I decided to keep the normal getCase flow untouched to make it easy to be used again later on. But I was not very convinced about this decision, as we will have this dead code here for a while. 
Do you think is better to remove the whole getCase functionality? The other option was to modify the current `getCase` funcion to call _/resolve_ inside, but I didn't like it either. What do you think would be the best approach?

Reply:
Oh I see. These are the test for the `use_get_case` not for the `getCase` API function. Sorry my mistake! I like your approach. Let's keep the `getCase` functionality for the moment as "dead code" and switch back when the resolve is not supported any more.